flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/ContextClassLoadingSettingTest.java

```diff
@@ -157,22 +161,26 @@ void testAkkaRpcService_ScheduleRunnableWithFixedRateSetsFlinkContextClassLoader
                                 contextClassLoaders.add(
                                         Thread.currentThread().getContextClassLoader());
                             } else {
-                                throw new RuntimeException("cancel task");
+                                terminalFuture.complete(null);
```

Comment:
You should still throw an exception such that the scheduled action stops early.

Reply:
updated.